neural_modelling/src/neuron/synapses.c

```diff
@@ -382,3 +382,28 @@ uint32_t synapses_get_pre_synaptic_events(void) {
     return (num_fixed_pre_synaptic_events +
             synapse_dynamics_get_plastic_pre_synaptic_events());
 }
+
+void synapses_flush_ring_buffers() {
```

Comment:
That should be `void synapses_flush_ring_buffers(void) {` because of legacy C weirdness. (You'd be OK in C++.)

Reply:
Good catch, somehow missed it here but not in the header file...